taichi/runtime/llvm/runtime.cpp

```diff
@@ -2,6 +2,10 @@
 // The generated bitcode will likely get inlined for performance.
 
 #if !defined(TI_INCLUDED) || !defined(_WIN32)
+// The latest MSVC uses llvm-11 as requirements. However, we use llvm-10
```

Comment:
A few nits:

* Add "NOTE(#2428)"
* Let's be explicit about "latest": Visual Studio 2019 version 16.10.1 (MSVC 14.29.30037)
* Add this link: https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/stl/inc/yvals_core.h#L550-L561
* guard it with `#if defined(_WIN32) || defined(_WIN64)`